sdk/identity/Azure.Identity/src/ChainedTokenCredential.cs

```diff
@@ -9,11 +9,11 @@
 
 namespace Azure.Identity
 {
-    public class AggregateCredential : TokenCredential
+    public class ChainedTokenCredential : TokenCredential
```

Comment:
I prefer `AggregateCredential`, personally.  I find it more descriptive and conceptually, more widely used.   The new name makes sense after looking at the implementation, but I wasn't understanding until I did so.